packages/react-components/components/Card.tsx

```diff
@@ -0,0 +1,24 @@
+import colors from '@celo/react-components/styles/colors.v2'
+import { elevationShadowStyle } from '@celo/react-components/styles/styles'
+import React from 'react'
+import { StyleSheet, View, ViewProps } from 'react-native'
+
+export interface Props extends ViewProps {
+  rounded?: boolean
+  children?: React.ReactNode
```

Comment:
when would use use without children?

Reply:
I don't know, mostly wanted the interface to be exactly like `<View />` 😉

Reply:
gotcha